docs/multi_vo_rucio.md

```diff
@@ -62,6 +62,56 @@ not be set for the server or the daemons as these parts of Rucio are not
 associated with a single VO. If `multi_vo` is not set, or set to False, then
 Rucio will operate normally.
 
+### Renew FTS cronjob
+
+Some daemons need a delegated X509 user proxy and the necessary CA so that it
+can submit jobs to FTS. For the CA you have to add a
+`<releasename>-rucio-ca-bundle` secret. For the user proxy a cronjob can be
+setup to either generate it from a long proxy. The cronjob uses the
```

Comment:
`either generate it from a long proxy`
suggests two methods, but only one described

Reply:
Fixed